tools/run_aten_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
This is a Travis test. Eventually, these tests will run on Jenkins, at which point we need to build with CUDA.